examples/beautify_javascript.py

```diff
@@ -0,0 +1,34 @@
+from libmproxy.protocol.http import decoded
+from libmproxy import version
+import jsbeautifier
+import sys
+import bs4
+import platform
+
+def start(context, argv):
+    if len(argv) != 1:
+        raise ValueError('Usage: -s "beautify_javascript"')
+
+def response(context, flow):
+    try:
+        header = '// beautified by jsbeautifier ' + jsbeautifier.__version__ + ' via mitmproxy ' + version.VERSION
+        server_header = '// running on Python ' + sys.version.replace('\n', '') + ' ' + (' '.join(platform.uname()))
```

Comment:
This would be needn't; But I added it for diagnostic purpose (There WAS multiple pythons on my mac....) and for fun (meh, this kind of comments ticks my nerd mentality, doesn't it?) :p